src/app/view-event/page.tsx

```diff
@@ -1,19 +1,100 @@
 'use client'
 import { UUID } from 'crypto'
-import EventView from '@/components/EventView'
+
 import { Suspense, useEffect, useState } from 'react'
-import EventCard from '@/components/EventCard'
 import { useSearchParams } from 'next/navigation'
 
+import EventView from '@/components/EventView'
+import EventCard from '@/components/EventCard'
+import Grid from '@/components/AvailabilityGrid'
+import Responses from '@/components/Responses'
+
+// Placeholder data for responders, including names and availability times
+interface Attendee {
+  user: { name: string }
+  timesegments: {
+    [key: string]: {
+      beginning: string
+      end: string
+      type: string
+    }[]
+  }
+}
+
+// gets attendee data from the API and formats it
+const getAttendeeData = (attendees: Attendee[]) => {
```

Comment:
I see this is with the mock data and you will be integrating with the api later.

Reply:
correct, should be updated in the next commit